backend/services/cloudkit_web_client.py

```
@@ -0,0 +1,387 @@
+            if hasattr(e, 'response') and e.response is not None:
+                logger.error(f"Response status: {e.response.status_code}")
+                logger.error(f"Response headers: {e.response.headers}")
+                logger.error(f"Response body: {e.response.text}")
+                
+                # Try to parse CloudKit error details
+                try:
+                    error_data = e.response.json()
+                    if 'errors' in error_data:
+                        for error in error_data['errors']:
+                            logger.error(f"CloudKit error: {error}")
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)